pa4/checker-pa4.ml

```diff
@@ -153,9 +164,9 @@ let rec get_super_classes input_class_name ast_list visited = begin
 		let ((cloc,_),inherits,_) = List.find(fun ((_,cname),_,_)-> input_class_name = cname) ast_list in
 		match inherits with
 		| None -> 
-			(*return object*) 
-			let class_object = ((cloc, "Object"), None, obj_features) in
-			[class_object];
+                        let obj_name = "Object" in
```

Comment:
Could you remove an indention level from this? It should only be single indented when compared to the indention on line 166. Having additional and inconsistent indention makes it harder to understand the logical structure of the program.